TA_Scheduler/utilities/AccountUtil.py

```diff
@@ -141,3 +141,19 @@ def getAccountByUsername(username: str) -> Optional[Account]:
             return None
         except IndexError:
             return None
+
+    def updateAccountInfo(
+        id: int, first: str, last: str, email: str, address: str, phone: int
+    ):
+        if account := AccountUtil.getAccountByID(id):
```

Comment:
walrus operator was introduced in python3.8, but Django supports python3.7, so while the test may pass on our machines, depending on what version of python we have, the test server will not pass as it test on python 3.7, 3.8, and 3.9

Reply:
Okay, I'll replace all of those.